ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/RestApiTest.java

```diff
@@ -344,6 +347,36 @@ public void shouldExecutePushQueryThatReturnsStreamOverWebSocketWithJsonContentT
     ));
   }
 
+  @Test
+  public void shouldExecutePullQueryThatReturnsStreamOverWebSocketWithJsonContentType() {
```

Comment:
Almost all the code is tested in the other endpoints, so we just have a quick smoke test here.

In the future, I hope we can refactor RQTT into a functional test suite that runs against all the endpoints.